ntbs-service/DataMigration/NotificationSearcher.cs

```diff
@@ -0,0 +1,186 @@
+                FROM Notifications n WHERE n.OldNotificationId = @NotificationId
+            )";
+
+        private readonly string connectionString;
+
+        public NotificationSearcher(IConfiguration _configuration)
+        {
+            connectionString = _configuration.GetConnectionString("migration");
+        }
+
+        public async Task<IEnumerable<Notification>> Search(string notificationId)
+        {
+            using (var connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+
+                var results = await connection.QueryAsync(Query, new
+                {
+                    NotificationId = $"{notificationId}"
+                });
+
+                return results.Select(AsNotification);
+            }
+        }
+
+        private static Notification AsNotification(dynamic result)
+        {
+            var notification = new Notification
+            {
+                ETSID = result.ETSID?.ToString(),
+                LTBRID = result.LTBRID?.ToString(),
+                NotificationDate = result.NotificationDate,
+                CreationDate = result.CreationDate,
+                PatientDetails = ExtractPatientDetails(result),
+                ClinicalDetails = ExtractClinicalDetails(result),
+                TravelDetails = ExtractTravelDetails(result),
+                VisitorDetails = ExtractVisitorDetails(result),
+                ComorbidityDetails = ExtractComorbidityDetails(result),
+                ImmunosuppressionDetails = ExtractImmunosuppressionDetails(result)
+            };
+
+            return notification;
+        }
+
+        private static ImmunosuppressionDetails ExtractImmunosuppressionDetails(dynamic notification)
+        {
+            return new ImmunosuppressionDetails {
+                Status = GetStatusFromString(notification.Status),
+                HasBioTherapy = GetBoolValue(notification.HasBioTherapy),
+                HasTransplantation = GetBoolValue(notification.HasTransplantation),
+                HasOther = GetBoolValue(notification.HasOther),
+                OtherDescription = notification.OtherDescription
+            };
+        }
+
+        private static ComorbidityDetails ExtractComorbidityDetails(dynamic notification)
+        {
+            return new ComorbidityDetails {
+                DiabetesStatus = GetStatusFromString(notification.DiabetesStatus),
+                LiverDiseaseStatus = GetStatusFromString(notification.LiverDiseaseStatus),
+                RenalDiseaseStatus = GetStatusFromString(notification.RenalDiseaseStatus),
+                HepatitisBStatus = GetStatusFromString(notification.HepatitisBStatus),
+                HepatitisCStatus = GetStatusFromString(notification.HepatitisCStatus)
+            };
+        }
+
+        private static ClinicalDetails ExtractClinicalDetails(dynamic notification)
+        {
+            return new ClinicalDetails
+                {

```

Comment:
Inconsistent braces used here I don't know if as a team we have a preference, but it should be consistent within the same file.

My preference:
```
return new ClinicalDetails
{
}
```

The two options you're using here:
```
return new ClinicalDetails {
}
```

```
return new ClinicalDetails 
    {
    }
```